htdocs/core/lib/security.lib.php

```diff
@@ -181,7 +181,7 @@ function dol_verifyHash($chain, $hash, $type='0')
  */
 function restrictedArea($user, $features, $objectid=0, $tableandshare='', $feature2='', $dbt_keyfield='fk_soc', $dbt_select='rowid', $objcanvas=null, $isdraft=0)
 {
-	global $db, $conf;
+	global $db, $conf, $user;
```

Comment:
Why adding $user here ? The user to test is already in first parameter.
If we need both (user running and user to test), we must rename the parameter $user into $fuser.

Reply:
@eldy 
yes you're right, i send a fix